pkg/network/ebpf/c/http.h

```diff
@@ -13,7 +14,7 @@ static __always_inline void http_prepare_key(u32 cpu, http_batch_key_t *key, htt
     key->page_num = batch_state->idx % HTTP_BATCH_PAGES;
 }
 
-static __always_inline void http_notify_batch(struct pt_regs *ctx) {
+static __always_inline void http_notify_batch(void *ctx) {
```

Comment:
Why change to a `void *` here?

Reply:
was for perf_output, but you can't use it in 4.4 (socket)
I will revert that